all.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "all",
     "summary": "Are all of the values true?",
-    "description": "Checks if **all** of the values in `data` are true. Evaluates all values from the first to the last element and stops once the outcome is unambiguous.\n\nIf only one value is given, the process evaluates to the given value. If no value is given (i.e. the array is empty) the process returns `null`.\n\nBy default all no-data values are ignored so that the process returns `null` if all values are no-data, `true` if all other values are true and `false` otherwise. Setting the `ignore_nodata` flag to `false` considers no-data values so that `null` is a valid logical object. If a component is `null`, the result will be `null` if the outcome is ambiguous. See the following truth table:\n\n```\n      || null  | false | true\n----- || ----- | ----- | -----\nnull  || null  | false | null\nfalse || false | false | false\ntrue  || null  | false | true\n```",
+    "description": "Checks if **all** of the values in `data` are true. Evaluates all values from the first to the last element and stops once the outcome is unambiguous. A result is ambiguous unless a value is `false`. If no value is given (i.e. the array is empty) the process returns `null`.\n\nBy default all no-data values are ignored so that the process returns `null` if all values are no-data, `true` if all values are true and `false` otherwise. Setting the `ignore_nodata` flag to `false` takes no-data values into account and the array values are reduced pairwise according to the following truth table:\n\n```\n      || null  | false | true\n----- || ----- | ----- | -----\nnull  || null  | false | null\nfalse || false | false | false\ntrue  || null  | false | true\n```",
```

Comment:
That is indeed already better.

One note though about  this in `all` description:

> A result is ambiguous unless a value is `false`.

I think this is a misleading statement: if all input values are `true`, the result is `true`, without ambiguity.

We could move the whole thing about ambiguity based short circuiting ("evaluates ... from first ... and stops once ... unambiguous ...") to the end as a implementation detail note (or drop it all together). For the user it is not that relevant after all. And in some implementations it might not be possible to stop as early as possible (e.g. because of  parallelization  aspects).

Reply:
Huh, there's something missing. 
My intention was to write: "A result is ambiguous unless a value is `false` or all values have been taken into account."

I agree that we should move it to the end.

Reply:
Hey @soxofaan,

sorry, it took me a while to get back to this. I moved the ambiguity details to the end and fixed the all description. Is this now ready to be merged?

Reply:
yes fine for me